apps/condo/pages/auth/change-password.tsx

```diff
@@ -133,10 +130,7 @@ const ChangePasswordPage: AuthPage = () => {
                     style={{ marginTop: '40px' }}
                     dependencies={['password']}
                     rules={[
-                        {
-                            required: true,
-                            message: PleaseConfirmYourPasswordMsg,
-                        },
+                        messageChanger(requiredValidator, PleaseConfirmYourPasswordMsg),
```

Comment:
Add declaration to validations as confirmPassword

Reply:
resolved 👍